src/aws/variables.tf

```diff
@@ -38,3 +38,8 @@ variable "medialive_lambda_name" {
   type    = string
   default = "marsha-medialive"
 }
+
+variable "create_shared_resources" {
```

Comment:
why not create resources only in production? It seems more robust at first thought. 
How do I know in which workspace this option was used?
What happens if I run `terraform apply` with the `create_shared_resources` flag set to true simultaneously in different workspaces? does it fail and nothing is broken?

Reply:
Yes this should be better but in fact the resources are not created first in production, but in `dev-*` workspace or staging. In production all this resources already exists.

Reply:
I completely rework the last commit to resolve this issue. I created a dedicated project for shared resources.

Reply:
So this variable should now be removed?

Reply:
yes